script/build.ts

```diff
@@ -247,8 +247,35 @@ function copyDependencies() {
   fs.mkdirpSync(gitDir)
   fs.copySync(path.resolve(projectRoot, 'app/node_modules/dugite/git'), gitDir)
 
+  if (process.platform === 'win32') {
+    console.log('  Cleaning unneeded Git components…')
+    const files = [
+      'Bitbucket.Authentication.dll',
+      'GitHub.Authentication.exe',
+      'Microsoft.Alm.Authentication.dll',
+      'Microsoft.Alm.Git.dll',
+      'Microsoft.IdentityModel.Clients.ActiveDirectory.Platform.dll',
+      'Microsoft.IdentityModel.Clients.ActiveDirectory.dll',
+      'Microsoft.Vsts.Authentication.dll',
+      'git-askpass.exe',
+      'git-credential-manager.exe',
+    ]
+
+    const gitCoreDir = path.join(gitDir, 'mingw64', 'libexec', 'git-core')
+
+    for (const file of files) {
+      const filePath = path.join(gitCoreDir, file)
+      try {
+        fs.statSync(filePath)
```

Comment:
Why do we bother stat'ing if we don't use the result?

Reply:
I used this to check for existence of the file but maybe `unlinkSync` will just fail in the same way. Either way, we don't really care about whether the file exists. I'll clean this up.